google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/internal/RowSetUtil.java

```diff
@@ -49,6 +49,45 @@
 public final class RowSetUtil {
   private RowSetUtil() {}
 
+  public static RowSet createSplitRanges(
+      RowSet rowSet, ByteString excludePoint, boolean fromStart) {
+    RowSet.Builder newRowSet = RowSet.newBuilder();
+
+    if (rowSet.getRowKeysList().isEmpty() && rowSet.getRowRangesList().isEmpty()) {
+      rowSet = RowSet.newBuilder().addRowRanges(RowRange.getDefaultInstance()).build();
```

Comment:
I don't think we should call `RowRange.getDefaultInstance()`? it should be     RowRange.newBuilder().setEndKeyOpen(excludePoint).build(); and RowRange.newBuilder().setStartKeyOpen(excludePoint).build();

so we're querying range (, largeRow) and (largeRow, )